src/styles/typography/index.md.njk

```diff
@@ -13,9 +13,14 @@ show_page_nav: true
 
 If your service is on the service.gov.uk subdomain you must use the GDS Transport font.
 
-You should use an alternative typeface like Helvetica or Arial for services that are publicly available on different domains.
+### When not to use the GDS Transport font
 
-If you’re not sure whether you should be using GDS Transport, read the Service Manual guide on [making your service look like GOV.UK](https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk) or [contact the Design System team](../../get-in-touch/).
+If your service is publicly available but on a different domain, use an alternative typeface like Helvetica or Arial.
```

Comment:
```suggestion
If your service is publicly available on a subdomain other than service.gov.uk, use an alternative typeface like Helvetica or Arial.
```

Reply:
@calvin-lau-sig7 That looks good, cheers. Will commit suggestion, squash commits and then ask for a final review. :)